kubernetes/templates/deployment.yaml

```diff
@@ -23,6 +23,13 @@ spec:
         runAsUser: {{ .Values.securityContext.runAsUser }}
         runAsGroup: {{ .Values.securityContext.runAsGroup }}
         fsGroup: {{ .Values.securityContext.fsGroup }}
+      initContainers:
+        - name: wait-for-postgresql
+          image: postgres
+          args:
+          - /bin/sh
+          - -c
+          - until pg_isready -h {{ .Values.config.db.host }} -p {{ .Values.config.db.port }} -U {{ .Values.config.db.port }}; do echo waiting for database; sleep 2; done;
```

Comment:
Shouldn't this be `-U {{ .Values.config.db.user }}` instead of `-U {{ .Values.config.db.port }}`?

Reply:
Oh yes. Surprised that my testing worked. Will push fix